grab.js

```diff
@@ -498,7 +494,7 @@ export class MoveGrab {
         Utils.later_add(Meta.LaterType.IDLE, () => {
             if (!global.display.end_grab_op && this.wasTiled) {
                 // move to current cursor position
-                let [x, y, _mods] = global.get_pointer();
+                let [x, y, _mods] = Utils.getPointerCoords();
```

Comment:
Likely unwarranted -- if we want touch coords to remain separate from a UX standpoint, this just won't cut it. Then again, I'm not even sure how that may relate to drag-and-drop softlock, so I'll wait until I get my code to work before I undo this change.